static/app/components/resolutionBox.tsx

```diff
@@ -17,23 +18,35 @@ import type {
 } from 'sentry/types/group';
 import {GroupActivityType} from 'sentry/types/group';
 import type {Repository} from 'sentry/types/integrations';
+import type {Organization} from 'sentry/types/organization';
+import type {Project} from 'sentry/types/project';
 
 type Props = {
-  projectId: string;
+  organization: Organization;
+  project: Project;
   // TODO(ts): This should be a union type `IgnoredStatusDetails | ResolvedStatusDetails`
   statusDetails: ResolvedStatusDetails;
   activities?: GroupActivity[];
 };
 
-function renderReason(
-  statusDetails: ResolvedStatusDetails,
-  projectId: string,
-  activities: GroupActivity[]
-) {
+export function renderResolutionReason({
+  statusDetails,
+  project,
+  organization,
```

Comment:
is there a reason we can't use `useOrganization` here?

Reply:
Didn't think it was worth refactoring the whole thing to be a hook. I'd have to look at the logic as well to ensure calling `useRenderResolutionReason` wouldn't cause issues if the issue wasn't resolved since hooks always need to be called and all that.

Passing a new prop skips all that